server.go

```diff
@@ -336,11 +338,7 @@ func (s *baseServer) handlePacket(p *receivedPacket) {
 	}()
 }
 
-func (s *baseServer) handlePacketImpl(p *receivedPacket) bool /* was the packet passed on to a session */ {
-	if len(p.data) < protocol.MinInitialPacketSize {
-		s.logger.Debugf("Dropping a packet that is too small to be a valid Initial (%d bytes)", len(p.data))
-		return false
-	}
+func (s *baseServer) handlePacketImpl(p *receivedPacket) bool /* was the packet handled */ {
```

Comment:
OT question: This function is called in a separate go routine above, so it may run in parallel for initial packets intended for the same session. Wouldn't we create two sessions in that case?

Reply:
https://github.com/lucas-clemente/quic-go/blob/54d9daba4468d719b086bfd37773b8e14f215d9c/server.go#L456-L462 is preventing this.

Reply:
This was resolved in #2299.